docs/component/col-picker.md

```
@@ -679,20 +923,28 @@ const columnChange = ({ selectedItem, resolve, finish }) => {
 | close    | 点击关闭按钮或者蒙层时触发 | -                                                | -        |
+| search   | 搜索时触发（filter-type="remote"） | `{ searchText, colIndex, selectedValues, selectedItems }` | - |
+| search-change | 搜索文本改变时触发（filter-type="remote"） | `{ searchText, colIndex, selectedValues, selectedItems }` | - |
+| search-clear | 清空搜索时触发（filter-type="remote"） | `{ searchText, colIndex, selectedValues, selectedItems }` | - |
+| refresh  | 下拉刷新时触发 | `{ searchText, colIndex, isRefresh: true }` | - |
+| scroll-to-lower | 上拉加载更多时触发 | `{ searchText, colIndex, isRefresh: false }` | - |
 
 ## Methods
 
 | 方法名称 | 说明             | 参数 | 最低版本 |
 | -------- | ---------------- | ---- | -------- |
 | open     | 打开 picker 弹框 | -    |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)